config/nas_mirror.sh

```
@@ -0,0 +1,84 @@
+        rm -rf "$DEST/$snap"
+        ((DELETED++))
+    fi
+done
+
+log "Deleted $DELETED obsolete snapshots"
+
+# ---- Copy snapshots missing in DEST ----
+COPIED=0
+FAILED=0
+while read -r src_snap; do
+    rel="${src_snap#$SOURCE/}"
```

Shellcheck (0.11.0)

[info] Expansions inside ${..} need to be quoted separately, otherwise they match as patterns.

(SC2295)